packages/hint-content-type/tests/tests.ts

```diff
@@ -231,6 +231,22 @@ const testsForDefaults: HintTest[] = [
             '/test.css': { headers: { 'Content-Type': 'text/css; charset=utf-8' } }
         }
     },
+    {
+        name: `Icon is served with 'Content-Type: image/vnd.microsoft.icon' header`,
+        reports: [],
```

Comment:
Nit: we generally omit the `reports` array entirely when there are no issues expected.